fs-one-concepts.md

```diff
@@ -3,19 +3,26 @@
 Try your best to answer each question on your own before looking up the answer online. Once you're done writing your first answer, you can google the question and write the best answer you find.
 
 #### 1. What is Enzyme and what are some of the methods that it provides?
-
+Enzyme is a testing framework for React, some of its methods are expect() toBeEqual() and so forth
```

Comment:
Close! Those are closer the Jest matchers. Try coming up with Enzyme methods (think of looking into and interacting with the components themselves.